pkg/data/domain/cluster/common.go

```diff
@@ -0,0 +1,90 @@
+package cluster
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/giantswarm/kubectl-gs/internal/key"
+)
+
+const (
+	defaultNamespace = "default"
+)
+
+func (s *Service) Get(ctx context.Context, options GetOptions) (runtime.Object, error) {
```

Comment:
This brings all the service methods together, so we can use a single method in the command runner